parser-typechecker/tests/Unison/Test/ClearCache.hs

```diff
@@ -0,0 +1,39 @@
+{-# Language OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+module Unison.Test.ClearCache where
+
+import Data.Foldable (for_)
+import Data.String.Here (i)
+import EasyTest
+import qualified Unison.Test.Ucm as Ucm
+import qualified Unison.Codebase as Codebase
+import qualified Unison.Var as WatchKind
+
+test :: Test ()
+test = scope "clearWatchCache" $
+  for_ [minBound .. maxBound] \fmt -> scope (show fmt) do
```

Comment:
A comment here about what `minBound` and `maxBound` are might help. I did a double take when I first read this because I had in mind integers or something, but you're just running the test for the old and new codebase format.